### PR TITLE
Update new instance types

### PR DESCRIPTION
**Pull request by jayanthvn**
**What type of PR is this?**
Unable to cherry pick the below PRs since  instance limits are auto generated.

**Which issue does this PR fix**:
#1219 
#1238 
#1289  

**What does this PR do / Why do we need it**:
Update instance type for 1.7.6 release

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:

**Testing done on this change**:

Yes

**Automation added to e2e**:

No

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
N/A

**Does this change require updates to the CNI daemonset config files to work?**:

**Does this PR introduce any user-facing change?**:

N/A

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by haouc — APPROVED** [review]
lgtm, thanks.

File: pkg/awsutils/vpc_ip_resource_limit.go [hunk 1]
```diff
@@ -144,7 +144,7 @@ var InstanceNetworkingLimits = map[string]InstanceTypeLimits{
 	"i3en.large":    {ENILimit: 3, IPv4Limit: 10},
 	"i3en.metal":    {ENILimit: 15, IPv4Limit: 50},
 	"i3en.xlarge":   {ENILimit: 4, IPv4Limit: 15},
-	"inf1.24xlarge": {ENILimit: 15, IPv4Limit: 30},
+	"inf1.24xlarge": {ENILimit: 11, IPv4Limit: 30},
```
Comment (fawadkhaliq) [inline pkg/awsutils/vpc_ip_resource_limit.go:147]: Curious why the limits changed for this instance type?

Comment (jayanthvn) [reply pkg/awsutils/vpc_ip_resource_limit.go:147]: Looks like https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html needs to be updated. This is automatically generated from aws cli describe instances - https://github.com/aws/amazon-vpc-cni-k8s/blob/615a2a6b6690abc02e043f13ebd16bb0f979be56/scripts/gen_vpc_ip_limits.go#L68

Comment (fawadkhaliq) [reply pkg/awsutils/vpc_ip_resource_limit.go:147]: Got it, thanks!

**Pull request merged**